pkg/controller/controller_binding_test.go

```diff
@@ -73,7 +73,7 @@ func TestReconcileServiceBindingNonExistingServiceInstance(t *testing.T) {
 	// There should only be one action that says it failed because no such instance exists.
 	updateAction := actions[0].(clientgotesting.UpdateAction)
 	if e, a := "update", updateAction.GetVerb(); e != a {
-		t.Fatalf("Unexpected verb on actions[0]; expected %v, got %v", e, a)
+		t.Fatalf("Unexpected verb on actions[0]; %s", expectedGot(e, a))
```

Comment:
This seems unrelated to the rest of this PR - am I missing something?

Reply:
I fixed this because I broke something in that test but the expected/got format was not updated and it was hard to debug so I fixed it as I went, but just this one. I can pull it from the PR if needed